documentation/high-avail/AMD-Pensando_HA_Proposal.md

```diff
@@ -50,16 +53,41 @@ Each ENI configured on the DPU is associated with a DP-VIP by the controller. In
 
 ### Datapath Heartbeat
 
-Each DPU sends heartbeat messages at a configured interval to its peer. When a peer loses a set number of heartbeats it declares the peer unreachable and a switchover is initiated. The interval between heartbeats and the number of missed heartbeats are configurable. The Heartbeats can be aggressive hence it is left to the DPU to perform the heartbeat.
+Each DPU sends heartbeat messages at a configured interval to its peer. When a peer loses a set number of heartbeats it declares the peer unreachable and a switchover is initiated. The interval between heartbeats and the number of missed heartbeats are configurable. The Heartbeats can be aggressive hence it is left to the DPU to perform the heartbeat. The Heartbeats can be aggressive hence it is left to the DPU to perform the heartbeat.
```

Comment:
Should we define the protocol used (UDP, BFD, other)?  Source / Dest ports?  If BFD performs multiphop probing, we could use this.  Need to think more on this...

Reply:
Also to check if multipath probing is implemented in SONiC BFD implementation. If HA inline sync utilizes any available path between active and standby (versus fixed path) then multipath probing would be required.